File: frontend/src/app/shared/models/feedback.ts
import { Response } from './response';
import { User } from './user';

export class Feedback {
  constructor(
    public id: number,
    public text: string,
    public createdAt: Date,
    public userId: number,
    public user: User,
    public responseId: number,
    public response: Response) {
  }
}

File: backend/Watcher.Common/Dtos/Feedback.cs
using System;
using Watcher.Common.MoveToFrontend;

namespace Watcher.Common.Dtos
{
    [ExportClassToTypescript]
    public class Feedback : Entity<int>
    {
        public override int Id { get; set; }
        public string Text { get; set; }
        public DateTime CreatedAt { get; set; }

        public int UserId { get; set; }
        public UserDto User { get; set; }

        public int? ResponseId { get; set; }
        public Response Response { get; set; }
    }
}
